### ISSUE TITLE
Improve web UI accessibility

### ISSUE THREAD

**Issue by miken32** [opnsense/core#367]
There's clearly been an attempt to modernise the web layout from what came before, but in a lot of ways it seems like one step forward, two steps back.

Clicking on firewall rules, for example, gives me a a vast white background with a bunch of black text and some icons. No indication of page structure, or clickability. If I tilt my laptop screen back so it's horizontal I can see that some of the page is a very light grey, suggesting that "Floating", "LAN", and "WAN" are tabs but that's about it for organization. If I hadn't been using pfSense for years I would have no idea what was going on.

As another example, compare the big bright "add a new schedule" button to the little black + button for adding a firewall rule.

The orange and black is a strong and distinctive colour scheme, let's see it used in the web interface! This isn't just an aesthetic issue; for users with limited visibility the almost complete lack of contrast can cause big accessibility issues. Ditto for the lack of clear functional delineations on the page.

**Comment by fichtner** [comment]
Firewall rules section is being reworked right now. If you can give feedback for opnsense-devel that would be great so we can weave it into the current development efforts:

https://forum.opnsense.org/index.php?topic=1305.0

Accessibility is a broader issue, requiring larger fonts, less intermediate colours and more contrast in general. It might be worth initiating a dedicated theme for that specific purpose?

**Comment by fichtner** [comment]
timeout
